test-clients/src/main/java/com/hedera/services/bdd/spec/infrastructure/HapiApiClients.java

```diff
@@ -137,7 +137,9 @@ private HapiApiClients(List<NodeConnectInfo> nodes, AccountID defaultNode) {
 		});
 		int after = stubCount();
 		this.defaultNode = defaultNode;
-		log.info("Constructed " + (after - before) + " new stubs building clients for " + this);
+		if (after > before) {
```

Comment:
Is this put in place to avoid these logs :

INFO   140  HapiApiClients - Constructed 0 new stubs building clients for HapiApiClients{node0=NodeConnectInfo{host=localhost, port=50211, tlsPort=50212, account=0.0.3}, default=0.0.3}
2020-08-10 11:39:30.830 INFO   140  HapiApiClients - Constructed 0 new stubs building clients for HapiApiClients{node0=NodeConnectInfo{host=localhost, port=50211, tlsPort=50212, account=0.0.3}, default=0.0.3}
2020-08-10 11:39:30.836 INFO   140  HapiApiClients - Constructed 0 new stubs building clients for HapiApiClients{node0=NodeConnectInfo{host=localhost, port=50211, tlsPort=50212, account=0.0.3}, default=0.0.3}
.
.
.

Reply:
Yes exactly :)